iwyu_output.h

```diff
@@ -321,6 +347,9 @@ class IwyuFileInfo {
   // Holds all the lines (#include and fwd-declare) that are reported.
   vector<OneIncludeOrForwardDeclareLine> lines_;
 
+  // Holds all of the using delcs that ar reported.
```

Comment:
It's probably some kind of potato, I was working from the comment on line 347 ("// Holds all the lines...") and figured it might make sense to keep them consistent :-)